import-export-cli/cmd/logout.go

```diff
@@ -49,10 +49,18 @@ var logoutCmd = &cobra.Command{
 }
 
 func runLogout(environment string) error {
+	cred, err := getCredentials(environment)
+	//Get current access token for
+	accessToken, err := credentials.GetOAuthAccessToken(cred, environment)
+	error := credentials.RevokeAccessToken(cred, environment, accessToken)
+	if error != nil {
+		return err
+	}
 	store, err := credentials.GetDefaultCredentialStore()
 	if err != nil {
 		return err
 	}
+	fmt.Println("Logged out from", environment, "environment")
```

Comment:
Don't we need to put a space at the end of the strings before environment?

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e